platform/ontology/model.go

```diff
@@ -41,3 +41,33 @@ type Block struct {
 	TxnList []Tx   `json:"TxnList"`
 	Hash    string `json:"Hash"`
 }
+
+type TxResponse struct {
```

Comment:
can you keep consistency with naming? `TxResponse` vs  `TransactionDetails` ?  two different styles